india_compliance/gst_india/setup/__init__.py

```diff
@@ -206,7 +206,7 @@ def _is_uom_mapped():
             (True for mapping in settings.gst_uom_mapping if mapping.uom == uom), False
         )
 
-    if not settings:
+    if not settings or settings.name != "GST Settings":
```

Comment:
settings should always be GST Settings. that is implied, isn't it?

Reply:
Implicit condition for setup complete.
I will make it explicit.